src/components/TypeFaceSubFamilies/_typeface-sub-families.scss

```
@@ -70,6 +70,23 @@
   padding-bottom: 2.5rem;
   margin-bottom: 0;
   display: flex;
+  font-size: calc(2rem + 0.25 * (100vw - 20rem) / 22);
+
+  @include carbon--breakpoint('md') {
+    font-size: calc(2.25rem + 0.375 * (100vw - 42rem) / 24);
+  }
+
+  @include carbon--breakpoint('lg') {
+    font-size: calc(2.625rem + 0.375 * (100vw - 66rem) / 16);
+  }
+
+  @include carbon--breakpoint('xlg') {
+    font-size: calc(3rem + 0.75 * (100vw - 82rem) / 17);
+  }
+
+  @include carbon--breakpoint('max') {
+    @include carbon--type-style('expressive-heading-05');
```

```suggestion
    @include carbon--type-style('expressive-heading-05', true);
```

We need this to get the correct responsive type size I believe.